config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, only: [:index, :show]
+
+  resources :posts, only: [:index, :show]
```

Comment:
- [ Required ] I think it would be better if you do as suggested by the previous reviewers, to satisfy the project requirements. The routes need to match the following patterns.

| Pages | urls |
|----------|-------|
|Users Index page| `/users/`|
|Users show page| `/users/:id/`|
|Posts index page| `/users/:user_id/posts/`|
|Posts show page| `/users/:user_id/posts/:id/`|

Currently, the routes in your application don't follow this pattern and that's why the previous reviewers requested the change. Please see the example below as a suggestion on how to achieve this.

## Example code
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
## Project requirements
![Screenshot from 2023-08-24 23-14-29](https://github.com/Cosmosha/blog-app/assets/16392046/3c379137-3175-4597-b2f1-af1296b7a688)